src/main/java/cl/transbank/webpay/example/controller/webpay/WebpayPlusMallController.java

```diff
@@ -74,10 +78,21 @@ public ModelAndView create(HttpServletRequest request) {
     }
 
     @RequestMapping(value = "/webpay_plus_mall/commit", method = { RequestMethod.GET, RequestMethod.POST })
-    public ModelAndView commit(@RequestParam("token_ws") String tokenWs, HttpServletRequest request) {
-        log.info(String.format("token_ws : %s", tokenWs));
+    public ModelAndView commit(HttpServletRequest request) {
 
+        String tokenWs = request.getParameter("token_ws");
         Map<String, Object> details = new HashMap<>();
+
+        if (tokenWs == null) {
```

Comment:
and undefined or empty string

Reply:
The changes will be made by adding the condition tokenWs.isEmpty() to the if statement.